vmware-esxi/Makefile

```diff
@@ -13,7 +13,7 @@ TIMEOUT ?= 1h
 
 all: vmware-esxi.dd.gz
 
-$(eval $(call check_packages_deps,fusefat,fusefat ))
+$(eval $(call check_packages_deps,fusefat,fusefat,mstools ))
```

Comment:
`check_packages_deps()` get two space-separated lists of packages as argument, one for >=22.04 and the other for Focal

Reply:
```suggestion
$(eval $(call check_packages_deps,fusefat mstools,fusefat mstools ))
```